lmms_eval/models/chat/thyme.py

```
@@ -0,0 +1,443 @@
+                    raise ValueError(f"Invalid message type: {s['type']}, {s}")
+                content.append(item)
+            new_inputs.append({"role": "user", "content": content})
+        return new_inputs
+
+    def _remove_unpickable_values(self, dictionary):
+        import pickle
+
+        def is_pickable(obj):
+            try:
+                pickle.dumps(obj)
+                return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)